plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from typing import Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from omegaconf import II
+
+
+@dataclass
+class RedisConf:
+    # host address
+    host: str = II("env:REDIS_HOST")
+    # port
+    port: int = II("env:REDIS_PORT")
+    # database, None for default
+    db: Optional[str] = II("env:REDIS_DB")
+    # password, None for no password
+    password: Optional[str] = II("env:REDIS_PASSWORD")
+    # if True, runs without redis server in single thread for testing purposes
+    mock: Optional[bool] = II("env:REDIS_MOCK")
```

Comment:
The env resolver will fail if the environment variables are not present.
you can add a default value if it makes sense:
```python
II("env:REDIS_HOST,localhost")
```